test/tax-lot/tax-lot.repository.mock.ts

```diff
@@ -2,22 +2,56 @@ import { generateMock } from "@anatine/zod-mock";
 import {
   findByBblRepoSchema,
   findByBblSpatialRepoSchema,
+  findZoningDistrictByTaxLotBblRepoSchema,
+  checkTaxLotByBblRepoSchema,
 } from "src/tax-lot/tax-lot.repository.schema";
 
 export class TaxLotRepositoryMock {
-  findByBblMocks = Array.from(Array(10), () =>
-    generateMock(findByBblRepoSchema, { seed: 1 }),
+  checkTaxLotByBblMocks = [
+    generateMock(checkTaxLotByBblRepoSchema, { seed: 1 }),
+  ];
+
+  async checkTaxLotByBbl(bbl: string) {
+    return this.checkTaxLotByBblMocks.find((row) => row.bbl === bbl);
+  }
+
+  findByBblMocks = Array.from(this.checkTaxLotByBblMocks, () =>
+    generateMock(findByBblRepoSchema, {
+      stringMap: {
+        bbl: () => this.checkTaxLotByBblMocks[0].bbl,
+      },
+    }),
   );
 
   async findByBbl(bbl: string) {
     return this.findByBblMocks.find((row) => row.bbl === bbl);
   }
 
-  findByBblSpatialMocks = Array.from(Array(10), () =>
-    generateMock(findByBblSpatialRepoSchema, { seed: 1 }),
+  findByBblSpatialMocks = Array.from(this.checkTaxLotByBblMocks, () =>
```

Comment:
Same `Array.from` -> `[array].map` feedback as above.

I also noticed that we swap from `checkTaxLotByBblMocks` to `findByBblMocks` in the reference. I think it makes sense to stick with the data from `checkTaxLotByBblMocks` throughout. The code sample above will satisfy this.

Reply:
Resolving since our discussions about potentially not wanting to use `checkTaxLotByBblMocks`